tools/manifest/download.py

```diff
@@ -15,7 +15,7 @@
 except ImportError:
     zstandard = None
 
-from .vcs import Git
+from .vcs import GitHasher as Git
```

Comment:
It feels very hacky to import this just to reuse the part to give a function for calling git. Can we factor that function out of the class?